### ISSUE TITLE
Make timeForLongPress configurable

### ISSUE THREAD

**Issue by shifenhutu** [hello-pangea/dnd#446]
### Description

The current value is set for quick dnd actions that are great for applications where dnd is a primary action which also seems to be the main focus of the library with regards to the examples. However, in my use case dnd is a secondary configuration action similarly to the homescreen on Android and the default value of 150ms is far to quick and easily causes items to be picked up when not intended to.

ref:

https://github.com/atlassian/react-beautiful-dnd/issues/415
https://github.com/atlassian/react-beautiful-dnd/pull/1304

**Comment by cadufeitosa** [comment]
Agree, this would be awesome for apps that do not have scroll in the page and the only feature is the dragging. App seems not responsive from user perspective.
